qvks/assignment1/Q1.py

```diff
@@ -0,0 +1,19 @@
+import string
+import sys
+
+# the function is case insensitive
+def is_anagram(s1, s2):
+	s1 = s1.lower()
+	s2 = s2.lower()
+	if len(s1) != len(s2):
+		return False
+
+	return sorted(s1)==sorted(s2)
+
+def main():
+	print(is_anagram("ab", "ba"))
```

Comment:
Can you use assert() to perform these checks? This way you can automatically verify your assertions.

Here are examples of using assert: https://www.tutorialspoint.com/python/assertions_in_python.htm